src/styles/_components/_inputs.scss

```diff
@@ -209,45 +208,77 @@ $checkbox-size: 32px;
     }
 }
 
-// BASIC CHECKBOX STYLES
 @mixin checkbox() {
-    @include flex();
-    @include align-items-center();
-    @include justify-content-center();
-    width: $checkbox-size;
-    height: $checkbox-size;
-    border: 1px solid color('input', 'border');
-    margin-right: 20px;
-    border-radius: 4px;
-    will-change: background-color;
-
-    svg {
-        display: none;
-        fill: color('primary');
-    }
+    display: none;
 
     & + label {
+        @include flex();
         height: $checkbox-size;
         line-height: $checkbox-size;
         @include font("IntervalNextReg");
-        font-size: em(18px);
+        font-size: em(16px);
         color: color('text', 'body');
-        will-change: color;
-    }
+        cursor: pointer;
+        position: relative;
 
-    &:focus,
-    &:checked,
-    &--checked {
-        border: 1px solid color('primary');
-        background-color: color('primary');
+        &:before {
+            content: "";
+            width: $checkbox-size;
+            height: $checkbox-size;
+            border: 2px solid color('input', 'border');
+            background: color('primary', 'highlight');
+            border-radius: 2px;
+            margin-right: 20px;
+        }
 
         svg {
-            display: block;
-            fill: color('background');
+            display: none;
+            position: absolute;
+            left: 9px;
+            top: calc(50% - 6px);
+            fill: color('primary', 'highlight');
         }
+    }
 
+    &:focus,
+    &:checked {
         & + label {
             color: color('primary');
+
+            svg {
+                display: block;
+            }
+
+            &:before {
+                background: color('primary');
+                border: 2px solid color('primary');
+                border-radius: 2px!important;
+            }
+        }
+    }
+
+    &--use-background {
+        &:focus,
+        &:checked {
+            & + label:before {
+                background: color('primary') url("../images/checkbox-icon.svg") center/65% no-repeat;
+            }
+        }
+    }
+
+    &--disabled {
+        + label {
+            pointer-events: none;
+            cursor: default;
+            opacity: 0.5;
```

Comment:
Instead of opacity, use the real colors. The colors are the same as the radio-button ones

```sass
'disabled-border': #f2f3f4,
'disabled-text': #bec7cb
```

Like this:
```sass
color: color('checkbox', 'disabled-text');
```